src/virus_clade_utils/get_clade_list.py

```diff
@@ -68,7 +69,8 @@ def get_clades(clade_counts: pl.LazyFrame, threshold: float, threshold_weeks: in
 
 # FIXME: provide ability to instantiate Config for the get_clade_list function and get the data_path from there
 def main(
-    genome_metadata_path: AnyPath = Config.nextstrain_latest_genome_metadata,
```

Comment:
Rather than download the metadata file from the URL listed on [Nextstrain's website](https://docs.nextstrain.org/projects/ncov/en/latest/reference/remote_inputs.html#remote-inputs-open-files), we'll download the file via an S3 https link.